newrelic-alerts-configurator/src/test/java/com/ocadotechnology/newrelic/alertsconfigurator/ConditionConfiguratorTest.java

```diff
@@ -90,6 +90,23 @@ public void shouldDoNothing_whenNoChannelsInConfiguration() {
         // when
         testee.sync(config);
 
+        // then
+        InOrder order = inOrder(alertsConditionsApiMock);
```

Comment:
`verifyZeroInteractions(alertsConditionsApiMock)` better conveys the intention. This applies to other tests classes for different condition types.